WebApp/WebContent/resources/export_postprocess/extract_temp_sal_flags.py

```diff
@@ -1,22 +1,24 @@
-def extract_temp_sal_flags(file_in,file_out):
-    ''' Finds error flags denoting bad intake temperature and salinity values, creates a corresponding boolean value in a separate column
-        file_in: original file
-	file_out: name of new file 
-	returns: file_out'''
-    import pandas as pd
-    file=file_in #'gofl20150410-sst_salinity_qc.csv'
-    df=pd.read_csv(file)
+import sys
+import pandas as pd
+''' Finds error flags denoting bad intake temperature and salinity values, creates a corresponding boolean value in a separate column'''
+try:
+    if len(sys.argv)==3:
```

Comment:
What happens if this isn't true? From the looks of it the script will just do nothing. It should really print a message saying what it expected.